camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -681,6 +681,12 @@ export default class Tasklist extends Mixins(TaskListMixin) {
     case "actionComplete":
       this.onFormSubmitCallback(customEvent.actionType);
       break;
+    default:
```

Comment:
i believe this change is already in master

Reply:
It was there in master, but not in dev